plugins/plugin-chart-echarts/src/Timeseries/transformProps.ts

```diff
@@ -144,7 +144,7 @@ export default function transformProps(chartProps: ChartProps): EchartsProps {
         top: 20,
         bottom: zoomable ? 80 : 20,
         left: 20,
-        right: 20,
+        right: showLegend && legendOrientation === LegendOrientation.Right ? 0 : 40,
```

Comment:
Could be nice to replace these magic numbers with constants. Let's rename `defaults.ts` to `constants.ts` and introduce a constant for this there.